test/controllers/declaration/WarehouseIdentificationControllerSpec.scala

```diff
@@ -46,34 +46,12 @@ class WarehouseIdentificationControllerSpec extends CustomExportsBaseSpec with W
       status(result) must be(OK)
     }
 
-    "display correct hints and questions" in {
```

Comment:
Have we agreed not to have these style tests?

Reply:
It's inside controller. This is covered in view spec

Reply:
Perhaps - I wasn't in the test meeting.  Can move to the view spec if required or remove?

Reply:
These should be in the `WarehouseIdentificationViewSpec.scala`, which I think we already have, so this code is redundant.
Looks good.

Reply:
It was testing code that was already in a ViewSpec